### PR TITLE
Support Kafka MT

### PR DESCRIPTION
**Pull request by thody**
- adds prefix to topic and consumer group

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -13,4 +13,5 @@ logging:
     com.heroku.kafka.demo: DEBUG
 
 kafka:
-  topic: ${KAFKA_TOPIC}
+  topic: ${KAFKA_PREFIX}messages
+  consumerGroup: ${KAFKA_PREFIX}demo
```
Comment (halorgium) [inline config.yml]: how about calling this something else?

Comment (thody) [inline config.yml]: Have a suggestion?

Comment (halorgium) [inline config.yml]: perhaps, something like "consumer-group"?

**Comment by halorgium** [comment]
:+1: :shipit:

**Pull request merged**